stdlib/importlib/abc.pyi

```diff
@@ -53,6 +50,17 @@ class PathEntryFinder(Finder):
     # Not defined on the actual class, but expected to exist.
     def find_spec(self, fullname: str, target: Optional[types.ModuleType] = ...) -> Optional[ModuleSpec]: ...
 
+class Loader(metaclass=ABCMeta):
+    def load_module(self, fullname: str) -> types.ModuleType: ...
+    def module_repr(self, module: types.ModuleType) -> str: ...
+    def create_module(self, spec: ModuleSpec) -> Optional[types.ModuleType]: ...
+    # Not defined on the actual class for backwards-compatibility reasons,
+    # but expected in new code.
+    def exec_module(self, module: types.ModuleType) -> None: ...
+
+class _LoaderProtocol(Protocol):
```

Comment:
Nit: According to `CONTRIBUTING`, this should probably be called `_SupportsLoadModule`, although in this case, `_LoaderProtocol` also is a good name.

Reply:
And thinking a bit more about it: Maybe we should add a `...Protocol` convention for protocols that mimic classes or abcs.